client/jetpack-cloud/sections/partner-portal/credit-card-fields/style.scss

```diff
@@ -0,0 +1,108 @@
+.credit-card-fields {
+	display: none;
+
+	&--is-loaded,
+	&--placeholder {
+		display: block;
+	}
+
+	&__input-field {
+		margin-top: 16px;
+
+		&:first-of-type {
+			margin-top: 0;
+		}
+
+		input {
+			padding: 11px 16px;
+			line-height: 0.7;
+			font-size: $font-body-small;
```

Comment:
There's no import at the top of the file which declares where `$font-body-small` is coming from. It works now but depending on which order this file is loaded, it might be changes because we "inherit" the latest defined version. Is it possible to find where it exists and at it at the top of the file?

Alternatively: might it be available as a CSS custom property instead since the rest of the files (although its just colours) uses CSS properties? 😄

Reply:
Nice catch! It looks we don't have the CSS variable available, but I have imported the file which declares the variable in 552c1e95e0fe85be9d8cec62d5a6940b6d523367